src/components/Stepper/index.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useEffect } from "react";
+import Button from "../Button";
+import Icon from "../Icon";
+
+const Stepper = ({
+  steps,
```

Comment:
Also I think there should be a way to pass and highlight errors on the step. I would say to discuss with UX on that